### PR TITLE
Fixed #30367 -- Changed "pip install" to "python -m pip install" in docs, comments and hints.

### PR DESCRIPTION
**Pull request by ramiro**
https://code.djangoproject.com/ticket/30367

### CODE REVIEW

File: docs/README [hunk 1]
```diff
@@ -6,7 +6,7 @@ This allows it to be built into other forms for easier viewing and browsing.
 
 To create an HTML version of the docs:
 
-* Install Sphinx (using ``pip install Sphinx`` or some other method)
+* Install Sphinx (using ``pthon -m pip install Sphinx`` or some other method)
```
Comment (sergeykolosov) [inline docs/README]: pthon → python

**Comment by rixx** [comment]
Apart from the typo, this looks good. Please unset the "patch needs improvement" flag on the ticket once you've fixed the typo.

File: docs/README (hunk 1 above)
Comment (ramiro) [reply docs/README]: Donde. Thanks!

**Comment by ramiro** [comment]
> You missed some:
> 
> ```
> docs/intro/whatsnext.txt:166:        $ pip install Sphinx
> ```
Yes, I somehow missed this one. Thanks!

> ```
> docs/releases/1.9.txt:1048:using pip, you can upgrade pip using ``pip install -U pip`` which will also
> ```
I didn't/don't want to touch release notes for past versions.

> ```
> django/db/models/fields/files.py:380:                          'or run command "pip install Pillow".'),
> django/test/runner.py:148:    pip install tblib
> tests/README.rst:5:    $ pip install -e ..
> tests/README.rst:6:    $ pip install -r requirements/py3.txt
> tests/invalid_models_tests/test_ordinary_fields.py:634:                      'or run command "pip install Pillow".'),
> tox.ini:3:# supported Python versions. To use it, "pip install tox" and then run "tox"
> ```
I limited myself to the `docs/` tree. Will work on these files located outside it. Thanks!

**Comment by pope1ni** [comment]
> I didn't/don't want to touch release notes for past versions.

Ok. I only mentioned it because things have been fixed up in release notes in the past, but it isn't essential.

**Comment by ramiro** [comment]
> > I didn't/don't want to touch release notes for past versions.
> 
> Ok. I only mentioned it because things have been fixed up in release notes in the past, but it isn't essential.

Thaks for the information! I've fixed it too now.

File: docs/man/django-admin.1 [hunk 2]
```diff
@@ -1916,7 +1916,7 @@ correctly:
 .sp
 .nf
 .ft C
-$ pip install tblib
```
Comment (felixxm) [inline docs/man/django-admin.1]: Man page is generated, so I will revert this change (see [django-admin-man-page](https://docs.djangoproject.com/en/stable/internals/contributing/writing-documentation/#django-admin-man-page)).

**Comment by felixxm** [comment]
@ramiro Thanks :+1:

**Pull request merged**